### PR TITLE
Update runtime version to 46

### PR DESCRIPTION
**Pull request by yakushabb**

### CODE REVIEW

**Comment by ghost** [comment]
@refi64 Could you review it.

File: fix_appdata.patch [hunk 1]
```diff
@@ -0,0 +1,42 @@
+diff --git a/share/appdata/howl.appdata.xml b/share/appdata/howl.appdata.xml
+index 5a131eb..6d9ae28 100644
+--- a/share/appdata/howl.appdata.xml
++++ b/share/appdata/howl.appdata.xml
+@@ -1,9 +1,12 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop">
+- <id>howl.desktop</id>
++ <id>io.howl.Editor</id>
```
Comment (bbhtt) [inline fix_appdata.patch]: You can drop this change, it is updated when using rename-appdata-file https://github.com/flathub/io.howl.Editor/blob/cffffcea4201ecc2df47052c5540bda116ccd211/io.howl.Editor.yaml#L6

File: fix_appdata.patch [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
++ </developer>
+  <summary>Lightweight editor with keyboard-centric minimalistic interface</summary>
+  <description>
+   <p>
+@@ -16,10 +19,17 @@
+  <screenshots>
+   <screenshot type="default">
+    <image width="1048" height="480">
+-    http://howl.io/images/screenshots/monokai/buffer-replace.png
++    https://howl.io/images/screenshots/monokai/buffer-replace.png
```
Comment (bbhtt) [inline fix_appdata.patch:20]: Seems nothing changed here?

File: fix_appdata.patch [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    </image>
+   </screenshot>
+  </screenshots>
+- <url type="homepage">http://howl.io</url>
+- <updatecontact>nino at nordman.org</updatecontact>
++ <content_rating type="oars-1.0" />
```
Comment (bbhtt) [inline fix_appdata.patch]:
```suggestion
+ <content_rating type="oars-1.1" />
```

File: fix_appdata.patch (hunk 2 above)
Comment (ghost) [reply fix_appdata.patch:20]: appstramcli does not like http connections. `s`

Comment (bbhtt) [reply fix_appdata.patch:20]: Aha indeed thanks, I didn't see the protocol change

**Comment by ghost** [comment]
@barthalion could you test it.

**Comment by refi64** [comment]
bot, build

**Comment by refi64** [comment]
thanks for this, sorry for the delay!

**Comment by refi64** [comment]
(it will probably be worth submitting the appdata changes upstream)

**Comment by ghost** [comment]
> (it will probably be worth submitting the appdata changes upstream)

It was merged last week https://github.com/howl-editor/howl/pull/614

**Pull request merged**